src/pages/ScriptPage.jsx

```diff
@@ -14,7 +14,7 @@ import ScriptInputOutputModal from "../components/sform/ScriptInputOutputModal.j
 import { Dropdown } from "semantic-ui-react";
 import { ICONS_MAP } from "../constants/dagreIcons.js";
 import ScriptFunctionSelection from "../components/ScriptFunctionSelection.jsx";
-import { Button } from "react-bootstrap";
+import { Button, Toast, ToastContainer } from "react-bootstrap";
```

Comment:
I do not see where `Toast, ToastContainer` are used.

Reply:
Moreover I believe that CSS style from https://github.com/kbss-cvut/s-pipes-editor-ui/blob/master/public/main.css#L2362-L2398 can be deleted as they were introduced by `toasted-notes` which is I believe not used anymore.

The relevant commit is a406715f2f008ecb0f6044fdf0ce82c5897cb6c2

Reply:
Toast and ToastContainer from bootstrap are used for displaying the alert. 

https://github.com/kbss-cvut/s-pipes-editor-ui/pull/105/commits/c418d8a6a5eb77cd269e3a74324624e0a8cb805b#diff-5790c2ee3d329fb66628f778c77059a400fd00a3142d9fa50360d703b6a44d55R631-R649